types/index.d.ts

```diff
@@ -55,6 +55,7 @@ export interface IConnectionOptions {
 export interface IChildConnectionOptions extends IConnectionOptions {
   url: string;
   appendTo?: HTMLElement;
+  iframe?: HTMLIFrameElement;
```

Comment:
It looks like `ERR_IFRAME_ALREADY_ATTACHED_TO_DOM` needs to be added to this file as well.

Reply:
You're right, it's done.

Reply:
I think it needs to be added at the bottom of the file like the other error constants as well.